web-frontend/modules/core/components/auth/TOTPLogin.vue

```diff
@@ -176,8 +175,31 @@ export default {
         this.$emit('success')
       } catch (error) {
         this.loadingVerifyBackupCode = false
-        const title = this.$t('totpLogin.verificationFailed')
+        this.handleError(error)
+      }
+    },
+    handleError(error) {
```

Comment:
Most of the error messaging are shown a toast/notification in the top right corner. Would it be possible to move them into the form as `<Alert>` on the following places?

<img width="1241" height="739" alt="Image" src="https://github.com/user-attachments/assets/19790d27-1fa5-4962-941e-465f94c2fda9" />

<img width="1241" height="739" alt="Image" src="https://github.com/user-attachments/assets/5a974905-f820-434c-b9e2-8502931e76c1" />

<img width="1247" height="739" alt="Image" src="https://github.com/user-attachments/assets/2c641384-f030-48ab-bce3-1fc4d82d01db" />